packages/components/src/Drawer/Drawer.component.js

```diff
@@ -80,25 +77,36 @@ export function cancelActionComponent(onCancelAction) {
 	if (!onCancelAction) {
 		return null;
 	}
-	const enhancedCancelAction = Object.assign({
-		icon: 'talend-cross',
-		hideLabel: true,
-		link: true,
-	}, onCancelAction);
+	const enhancedCancelAction = Object.assign(
+		{
+			icon: 'talend-cross',
+			hideLabel: true,
+			link: true,
+		},
+		onCancelAction,
+	);
 	return <Action className={theme['tc-drawer-close-action']} {...enhancedCancelAction} />;
 }
 
-function DrawerTitle({ title, children, onCancelAction }) {
+export function subtitleComponent(subtitle) {
+	if (!subtitle || subtitle === '') {
+		return null;
+	}
+	return <h2 title={subtitle}>{subtitle}</h2>;
+}
+
+function DrawerTitle({ title, subtitle, children, onCancelAction }) {
 	if (!title) {
 		return null;
 	}
 	return (
-		<div className={theme['tc-drawer-header']}>
-			<div className={theme['tc-drawer-header-title']}>
+		<div className={`tc-drawer-header ${theme['tc-drawer-header']}`}>
```

Comment:
Could you change element by `header` ?
(I know it's not the purpose of the pull-request)